### PR TITLE
Bump version from 2.9.2 to 2.9.3

### PR DESCRIPTION
**Pull request by rynewang**
Bump version from 2.9.2 to 2.9.3

Note: previous PRs did not update doc/source/ray-overview/pip_freeze_ray-py39-cpu.txt and doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt. But I feel it worths updating.

### CODE REVIEW

**Review by aslonnie — COMMENTED** [review]
please reference this PR to make the changes:

https://github.com/ray-project/ray/pull/42719/files

only a small number of files require changes. the step is wrong, and we are changing it into another script. The new script is already merged on master, but not on this branch.

**Comment by GeneDer** [comment]
@aslonnie do you know if 2.9.3 was cut from master or 2.9.2?

**Comment by GeneDer** [comment]
oh, I see. The command should be running to change 2.9.2 to 2.9.3.

**Comment by rynewang** [comment]
@aslonnie @GeneDer this PR is force-pushed updated. Please check. Please also help figure out if `doc/source/ray-overview/pip_freeze_ray-py39-cpu.txt` and `doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt` changes make sense. Thanks.

File: doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt [hunk 1]
```diff
@@ -355,8 +355,8 @@ PyWavelets==1.4.1
 PyYAML==6.0.1
 pyzmq==24.0.1
 querystring-parser==1.2.4
-ray @ file:///home/ray/ray-2.9.2-cp39-cp39-manylinux2014_x86_64.whl#sha256=7e073e8f1c465064c40c9318376b5bd4568589f2c7ae4e3bc5f86a4e7b0b2fe0
-ray-cpp==2.9.2
+ray @ file:///home/ray/ray-2.9.3-cp39-cp39-manylinux2014_x86_64.whl#sha256=7e073e8f1c465064c40c9318376b5bd4568589f2c7ae4e3bc5f86a4e7b0b2fe0
+ray-cpp==2.9.3
```
Comment (GeneDer) [inline doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt:359]: I think those pip freeze files are added when we update the docker dependencies in  this [release step](https://www.notion.so/anyscale-hq/How-to-release-Ray-go-release-ray-28e4a2347577471288bbd7d64001fa88?pvs=4#9cc39bb28f5742778b62dd1381601e03) For now let's take it out. Same for the other one

**Review by GeneDer — COMMENTED** [review]
Let's take out the pip freeze file changes. But otherwise LGTM!

File: doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt (hunk 1 above)
Comment (rynewang) [reply doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt]: removed, thanks!

**Review by GeneDer — APPROVED** [review]
:shipit:

**Review by aslonnie — APPROVED** [review]

**Comment by aslonnie** [comment]
I am force merging this for you.

the min install tests are bad due to something else.

not sure about the failing windows tests though.

**Pull request merged**